packages/tx-construction/src/tx-builder/OutputBuilder.ts

```diff
@@ -21,6 +21,8 @@ export interface OutputBuilderProps {
   txOut?: PartialTxOut;
   /** Logger */
   logger: Logger;
+  /** Handle Provider for ADAHandle */
```

Comment:
```suggestion
  /** Handle Provider for resolving addresses */
```

Reply:
e25a4e19c0746a361b37 this should solve this